### PR TITLE
Feature/2007/create cf deploy config writer

### PR DESCRIPTION
**Pull request by longieirl**
Fix for https://github.com/SAP/open-ux-tools/issues/2007

- Introduce a new cf-deploy-config-writer, will be used CF deployment flows to support the following;
a. Add CF deployment config to an existing HMTL5 application
b. To generate a base CF config i.e. standalone or managed
- `MtaConfig` class will provide a wrapper for the `@sap/mta-lib` to edit `mta.yaml` files used in the CF generation flow.

### CODE REVIEW

File: .changeset/honest-poets-sniff.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@sap-ux/cf-deploy-config-writer': patch
+'@sap-ux/project-access': patch
```
Comment (devinea) [inline .changeset/honest-poets-sniff.md:3]:
```suggestion
```
no project-access changes

File: .changeset/honest-poets-sniff.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+'@sap-ux/ui5-config': patch
+'@sap-ux/telemetry': patch
```
Comment (devinea) [inline .changeset/honest-poets-sniff.md:5]:
```suggestion
'@sap-ux/telemetry': patch
```
telemetry are only tests

**Review by devinea — COMMENTED** [review]
New module structure set-up looks good. 
ui5-config changes look good. 
changeset should be updated. 
I haven't reviewed the cf-deploy-config-writer logic in detail, but it looks clear and well structured.

File: .changeset/honest-poets-sniff.md (hunk 2 above)
Comment (longieirl) [reply .changeset/honest-poets-sniff.md:5]: Done.

File: .changeset/honest-poets-sniff.md (hunk 1 above)
Comment (longieirl) [reply .changeset/honest-poets-sniff.md:3]: Done

**Review by cianmSAP — APPROVED** [review]
Code is well structured and documented
Coverage > 90%
Changeset ✅

**Review by cianmSAP — APPROVED** [review]
Re-approval

**Review by IainSAP — APPROVED** [review]
Thx @longieirl 

- Not tested locally
- Coverage is good

**Review by devinea — APPROVED** [review]
New module step looks good and covered by tests. 
ui5 config changes lg.
Changeset ✅

**Pull request merged**